<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentUnboxingWizard</class>
 <widget class="QWidget" name="AgentUnboxingWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>336</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>409</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonNextStep">
     <property name="text">
      <string>UNBOX</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>408</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>337</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="LightWidget" name="lightWidgetPaired" native="true">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="LightWidget" name="lightWidgetControllerSet" native="true">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="LightWidget" name="lightWidgetControllerConfigured" native="true">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="LightWidget" name="lightWidgetAllDone" native="true">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelControllerConfigured">
       <property name="text">
        <string>Controller configured</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="LightWidget" name="lightWidgetDelivered" native="true">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="IdenticonLabel" name="labelAgentName">
       <property name="text">
        <string>Agent name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelDelivered">
       <property name="text">
        <string>Delivered</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelControllerName">
       <property name="text">
        <string>Controller name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labelPaired">
       <property name="text">
        <string>All done</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelPaired_2">
       <property name="text">
        <string>Paired</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelControllerSet">
       <property name="text">
        <string>Controller set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LightWidget</class>
   <extends>QWidget</extends>
   <header location="global">components/LightWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IdenticonLabel</class>
   <extends>QLabel</extends>
   <header location="global">identity/IdenticonLabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonNextStep</sender>
   <signal>clicked()</signal>
   <receiver>AgentUnboxingWizard</receiver>
   <slot>nextStageClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>nextUnboxingStep()</signal>
  <slot>nextStageClicked()</slot>
 </slots>
</ui>
